notebooks/wikiqa/arci.ipynb

```diff
@@ -0,0 +1,749 @@
+   "metadata": {},
+   "source": [
+    "#### Download\n",
+    "\n",
+    "We first download the GLoVe pre-trained word embeddings. Be pationt - this takes a long time related to your network service. Alternatively, you can download it via another downloading tool and put **glove.840B.300d.txt** in the directory **../../data/embedding**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 50,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2018-10-20 20:20:26--  http://nlp.stanford.edu/data/glove.840B.300d.zip\n",
+      "正在解析主机 nlp.stanford.edu (nlp.stanford.edu)... 171.64.67.140\n",
+      "正在连接 nlp.stanford.edu (nlp.stanford.edu)|171.64.67.140|:80... 已连接。\n",
+      "已发出 HTTP 请求，正在等待回应... 302 Found\n",
+      "位置：https://nlp.stanford.edu/data/glove.840B.300d.zip [跟随至新的 URL]\n",
+      "--2018-10-20 20:20:27--  https://nlp.stanford.edu/data/glove.840B.300d.zip\n",
+      "正在连接 nlp.stanford.edu (nlp.stanford.edu)|171.64.67.140|:443... 已连接。\n",
+      "已发出 HTTP 请求，正在等待回应... 200 OK\n",
+      "长度：2176768927 (2.0G) [application/zip]\n",
+      "正在保存至: “../../data/embedding/glove.840B.300d.zip.5”\n",
+      "\n",
+      "glove.840B.300d.zip 100%[===================>]   2.03G   535KB/s  用时 52m 54s   \n",
+      "\n",
+      "2018-10-20 21:13:22 (670 KB/s) - 已保存 “../../data/embedding/glove.840B.300d.zip.5” [2176768927/2176768927])\n",
+      "\n",
+      "Archive:  ../../data/embedding/glove.840B.300d.zip\n",
+      "  End-of-central-directory signature not found.  Either this file is not\n",
+      "  a zipfile, or it constitutes one disk of a multi-part archive.  In the\n",
+      "  latter case the central directory and zipfile comment will be found on\n",
+      "  the last disk(s) of this archive.\n",
+      "unzip:  cannot find zipfile directory in one of ../../data/embedding/glove.840B.300d.zip or\n",
+      "        ../../data/embedding/glove.840B.300d.zip.zip, and cannot find ../../data/embedding/glove.840B.300d.zip.ZIP, period.\n"
+     ]
+    }
+   ],
+   "source": [
+    "!mkdir -p ../../data/embedding\n",
+    "!wget -O ../../data/embedding/glove.840B.300d.zip http://nlp.stanford.edu/data/glove.840B.300d.zip \n",
+    "!unzip -o ../../data/embedding/glove.840B.300d.zip -d ../../data/embedding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Preproceesing with pre-trained embeddings\n",
+    "\n",
+    "We run the processor in three lines of code.  \n",
+    "If we use **the randomly initialized embeddings**, just keep default value for *embedding_file*, and the preprocessing code is as follows:\n",
+    "```python\n",
+    "from matchzoo import preprocessor\n",
+    "arci_preprocessor = preprocessor.ArcIPreprocessor()\n",
+    "processed_tr = arci_preprocessor.fit_transform(train, stage='train')\n",
+    "processed_te = arci_preprocessor.fit_transform(test, stage='test')\n",
```

Comment:
Here, the `state`  should be `evaluate` as it is to evaluate the performance of test set.